gapic/src/main/com/google/gapid/image/FetchedImage.java

```diff
@@ -94,10 +94,12 @@
     });
   }
 
-  public static ListenableFuture<ImageData> loadLevel(
-      ListenableFuture<FetchedImage> futureImage, final int level) {
+  public static ListenableFuture<ImageData> loadImage(
+      ListenableFuture<FetchedImage> futureImage, final int layer, final int level) {
     return Futures.transformAsync(futureImage, image -> Futures.transform(
-        image.getLevel(Math.min(level, image.getLevelCount())), (l) -> l.getImageData()));
+        image.getImage(
+            Math.min(layer, image.getLayerCount()),
+            Math.min(level, image.getLevelCount())), (l) -> l.getImageData()));
```

Comment:
`getLayerCount() - 1` and `getLevelCount() - 1`, no?

Reply:
Good spot - bug was carried over from before this PR.